docker/amdvlk.Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN export DEBIAN_FRONTEND=noninteractive && export TZ=America/New_York \
     && apt-get update \
     && TOOLCHAIN_PACKAGES="gcc g++ binutils-gold" \
     && if echo "$FEATURES" | grep -q "+clang" ; then \
-         TOOLCHAIN_PACKAGES="clang-9 libclang-common-9-dev lld-9"; \
+         TOOLCHAIN_PACKAGES="clang-9 libclang-common-9-dev lld-9 clang-tidy"; \
```

Comment:
Can we install a fixed clang-tidy version? This will make it easier to reproduce locally.